.eslintrc.js

```diff
@@ -21,5 +21,7 @@ module.exports = {
   plugins: ['react', '@typescript-eslint', 'prettier'],
   rules: {
     'react/react-in-jsx-scope': 0,
+    'import/prefer-default-export': 0,
+    '@typescript-eslint/no-non-null-assertion': 0,
```

Comment:
What does this do?

Reply:
In Users.ts we are passing this:-  
```
const streamChat = StreamChat.getInstance(
  process.env.STREAM_API_KEY!, 
  process.env.STREAM_PRIVATE_API_KEY
);
```
process.env.STREAM_API_KEY and we are putting ! behind this to tell the server that this value is defined somewhere and it won't be null. But in esLint it is forbidden to use (!) for this reason. And it was giving error so I fixed it by using 
```
'@typescript-eslint/no-non-null-assertion': 0,
```

Reply:
Ok